### PR TITLE
Fragments Pallet

### PR DESCRIPTION
**Pull request by sinkingsugar**

### CODE REVIEW

File: pallets/detach/src/lib.rs [hunk 1]
```diff
@@ -161,7 +161,7 @@ pub mod pallet {
 
 	#[pallet::storage]
 	pub type DetachNonces<T: Config> =
-		StorageDoubleMap<_, Blake2_128Concat, Vec<u8>, Blake2_128Concat, SupportedChains, u64>;
+		StorageDoubleMap<_, Twox64Concat, Vec<u8>, Twox64Concat, SupportedChains, u64>;
```
Comment (aang114) [inline pallets/detach/src/lib.rs:164]: Why did you change `Blake2_128Concat` to `Twox64Concat` btw? @sinkingsugar

Comment (sinkingsugar) [reply pallets/detach/src/lib.rs:164]: Smaller footprint. Enough security.
